Description: Use memcache session engine by default
 This is to avoid frequent log outs, as sessions are not shared across workers.
Author: Dave Walker (Daviey) <DaveWalker@ubuntu.com>
Bug: https://launchpad.net/bugs/968850

Index: horizon/openstack_dashboard/local/local_settings.py.example
===================================================================
--- horizon.orig/openstack_dashboard/local/local_settings.py.example	2012-07-03 14:07:47.016119233 -0700
+++ horizon/openstack_dashboard/local/local_settings.py.example	2012-07-03 14:07:58.952119749 -0700
@@ -34,10 +34,7 @@
 # from horizon.utils import secret_key
 # SECRET_KEY = secret_key.generate_or_read_from_file(os.path.join(LOCAL_PATH, '.secret_key_store'))
 
-# We recommend you use memcached for development; otherwise after every reload
-# of the django development server, you will have to login again. To use
-# memcached set CACHE_BACKED to something like 'memcached://127.0.0.1:11211/'
-CACHE_BACKEND = 'locmem://'
+CACHE_BACKEND = 'memcached://127.0.0.1:11211/'
 
 # Send email to the console by default
 EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
